pkg/reporting/app.go

```diff
@@ -327,3 +330,19 @@ func getDownstreamInfo(appID string) (*types.DownstreamInfo, error) {
 
 	return &di, nil
 }
+
+func getGitOpsReport(clientset kubernetes.Interface, appID string, clusterID string) *types.GitOpsReport {
+	gitOpsConfig, err := gitops.GetDownstreamGitOpsConfig(clientset, appID, clusterID)
+	if err != nil {
+		logger.Debugf("failed to get gitops config: %v", err.Error())
+		return nil
+	}
+
+	if gitOpsConfig != nil {
+		return &types.GitOpsReport{
+			Provider:  gitOpsConfig.Provider,
+			IsEnabled: gitOpsConfig.IsConnected,
```

Comment:
do we care if it's connected or not? isn't the criteria here to report if gitops is configured? if so, we might need to change this.